arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -385,7 +359,7 @@
             "schema": {
               "$ref": "#/definitions/StorageAccountRegenerateKeyParameters"
             },
-            "description": "Specifies name of the key which should be regenerated. key1 or key2 for the default keys"
+            "description": "Specifies name of the key which should be regenerated -- key1 or key2."
```

Comment:
User can not have other names other than key1 or key2 in this api-version, in the latest swagger this object model has got changed completely